notebooks/official/matching_engine/intro-swivel.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if not os.getenv("IS_TESTING"):
</pre><p>Why are you exempting it from tests?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>